src/main/java/Task.java

```diff
@@ -0,0 +1,24 @@
+import java.util.*;
+
+public class Task {
+    public static final String descriptionPrefix = "description: ";
+    private String description;
+    private boolean important;
```

Comment:
should it explain the named entity to the reader accurately and at a sufficient level of detail? boolean important, should it changed to isImportant ?